src/python_testing/TC_CADMIN_1_5.py

```diff
@@ -44,13 +47,63 @@
 
 
 class TC_CADMIN_1_5(MatterBaseTest):
-    async def get_txt_record(self):
+    @dataclass
+    class ParsedService:
+        service: mdns_discovery.MdnsServiceInfo
+        cm: Optional[int] = None
+        d: Optional[int] = None
+
+        def __post_init__(self):
+            self.cm = int(self.service.txt_record.get('CM', None))
```

Comment:
I think you should do some try/except because if you do miss CM you will have:

```
>>> int(None)                                                                                                                                                                     Traceback (most recent call last):                                                                                                                                              
File "<stdin>", line 1, in <module>                                                                                                                                            TypeError: int() argument must be a string, a bytes-like object or a real number, not 'NoneType'
```